src/stream/src/executor/source/list_executor.rs

```diff
@@ -78,38 +76,28 @@ impl<S: StateStore> FsListExecutor<S> {
             .map_err(StreamExecutorError::connector_error)?
             .map_err(StreamExecutorError::connector_error);
 
-        // Group FsPageItem stream into chunks of size 1024.
-        let chunked_stream = stream.chunks(CHUNK_SIZE).map(|chunk| {
-            let rows = chunk
-                .into_iter()
-                .map(|item| match item {
-                    Ok(page_item) => Ok((
-                        Op::Insert,
-                        OwnedRow::new(vec![
-                            Some(ScalarImpl::Utf8(page_item.name.into_boxed_str())),
-                            Some(ScalarImpl::Timestamptz(page_item.timestamp)),
-                            Some(ScalarImpl::Int64(page_item.size)),
-                        ]),
-                    )),
-                    Err(e) => {
-                        tracing::error!(error = %e.as_report(), "Connector fail to list item");
-                        Err(e)
-                    }
-                })
-                .collect::<Vec<_>>();
-
-            let res: Vec<(Op, OwnedRow)> = rows.into_iter().flatten().collect();
-            if res.is_empty() {
-                tracing::warn!("No items were listed from source.");
-                return Ok(StreamChunk::default());
+        let processed_stream = stream.map(|item| match item {
+            Ok(page_item) => {
+                let row = (
```

Comment:
Why do we change to 1 file 1 chunk here? :eyes:
The name `chunked_paginate_stream` is confusing

Reply:
make sense, why?
IIRC, before migrating to opendal, the s3 sdk returns a paginated list. I think the func name retained from then.